### PR TITLE
Release PR with fixes for issues 454, 455

### LINKED ISSUE #454: OSDP resource is missing in API1800
### Scenario/Intent
[What you are trying to achieve but can't?]
OSDP resource is not supported in API1800

### Environment Details
 - **OneView SDK Version:** [Version of this SDK for which you are encountering the issue]
 - **OneView Appliance Version:** [Version of the OneView appliance you're interacting with]
 - **OneView Client API Version:** [API version of your client object]
 - **Ruby Version:** [Version of Ruby in your environment]
 - **Platform:** [OS distribution and release version]

### Steps to Reproduce
[What are the complete steps needed to do in order to reproduce your problem?]

### Expected Result
[What do you *expect* to happen after taking the steps above?]

### Actual Result
[What *actually* happens after the steps above? Include error output or a link to a gist.]

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Issues Resolved
Fixes #454, #455 

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api1800/synergy/os_deployment_plan.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api300/synergy/os_deployment_plan'
+
+module OneviewSDK
+  module API1800
+    module Synergy
+      # OS Deployment Plan resource implementation for API1800 Synergy
+      class OSDeploymentPlan < OneviewSDK::API300::Synergy::OSDeploymentPlan
```
Comment (yuvirani) [inline lib/oneview-sdk/resource/api1800/synergy/os_deployment_plan.rb:18]: why are we inheriting from 300 and not from 1600?

Comment (VenkateshRavula) [reply lib/oneview-sdk/resource/api1800/synergy/os_deployment_plan.rb]: changed the parent class

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## v5.15.0
```
Comment (chebroluharika) [inline CHANGELOG.md:1]: keep unreleased beside this. If this is a patch, then version should be v5.14.1

File: lib/oneview-sdk/resource/api1800/synergy/server_profile.rb [hunk 3]
```diff
@@ -9,13 +9,16 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
+require_relative '../../api300/synergy/server_profile'
 require_relative '../c7000/server_profile'
```
Comment (chebroluharika) [inline lib/oneview-sdk/resource/api1800/synergy/server_profile.rb:13]: why do we need this?

File: CHANGELOG.md (hunk 2 above)
Comment (VenkateshRavula) [reply CHANGELOG.md:1]: We support new resources in this release. So this is minor release, not a patch

**Pull request merged**